src/components/SweetsList/SweetsList.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from "../../common/test-utils";
+import axios from "axios";
+import SweetsList from "./SweetsList";
+import { mockedSweets } from "../../common/test-data";
+
+//setting up axios
+// jest.mock("axios");
+
+describe("SweetsList component", () => {
+  test("render sweets if request succeed", async () => {
+    // const resp = {
+    //   data: mockedSweets,
+    // };
+    // (axios.get as jest.Mock).mockResolvedValueOnce(resp);
+    render(<SweetsList />);
+    const sweetItems = await screen.findAllByRole("listitem");
+    expect(sweetItems).toHaveLength(1);
```

Comment:
In addition to checking if we got the list, it's also a good idea to try to check if we actually have anything in the DOM. So like trying to find an element by text (like a title or candy name) etc. and assert that it is in the document